### PR TITLE
removes the ambience from the supermatter

### PR DESCRIPTION
**Pull request by ShadowDeath6**
not a fan of having tinnitus, thanks

:cl: ShadowDeath6
sounddel: Removed ambient ringing from supermatter shard.
/:cl:

### CODE REVIEW

**Review by GrayRachnid — CHANGES_REQUESTED** [review]
I feel retarded, just comment it out.

**Review by GrayRachnid — APPROVED** [review]
inb4 nich comes in and says no you retard remove it or modularize it but i don't deserve to have maintainer for this shit so i'm spewing crap outta my ass

**Comment by ShadowDeath6** [comment]
:pensive:

File: code/datums/looping_sounds/machinery_sounds.dm [hunk 1]
```diff
@@ -8,10 +8,10 @@
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-/datum/looping_sound/supermatter
+/*/datum/looping_sound/supermatter
```
Comment (ThatLing) [inline code/datums/looping_sounds/machinery_sounds.dm]: dont comment this out

File: code/modules/power/supermatter/supermatter.dm [hunk 2]
```diff
@@ -139,7 +139,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 	var/is_main_engine = FALSE
 
-	var/datum/looping_sound/supermatter/soundloop
+	//var/datum/looping_sound/supermatter/soundloop
```
Comment (ThatLing) [inline code/modules/power/supermatter/supermatter.dm]: Please mark your changes as explained here

File: code/modules/power/supermatter/supermatter.dm [hunk 3]
```diff
@@ -158,7 +158,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	if(is_main_engine)
 		GLOB.main_supermatter_engine = src
 
-	soundloop = new(list(src), TRUE)
+	//soundloop = new(list(src), TRUE)
```
Comment (ThatLing) [inline code/modules/power/supermatter/supermatter.dm]: Please mark your changes as explained here

File: code/modules/power/supermatter/supermatter.dm [hunk 4]
```diff
@@ -168,7 +168,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	QDEL_NULL(countdown)
 	if(is_main_engine && GLOB.main_supermatter_engine == src)
 		GLOB.main_supermatter_engine = null
-	QDEL_NULL(soundloop)
+	//QDEL_NULL(soundloop)
```
Comment (ThatLing) [inline code/modules/power/supermatter/supermatter.dm]: Please mark your changes as explained here

File: code/modules/power/supermatter/supermatter.dm [hunk 5]
```diff
@@ -304,8 +304,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	if(!istype(T)) 	//We are in a crate or somewhere that isn't turf, if we return to turf resume processing but for now.
 		return  //Yeah just stop.
 
-	if(power)
-		soundloop.volume = min(40, (round(power/100)/50)+1) // 5 +1 volume per 20 power. 2500 power is max
+	//if(power)
+		//soundloop.volume = min(40, (round(power/100)/50)+1) // 5 +1 volume per 20 power. 2500 power is max
```
Comment (ThatLing) [inline code/modules/power/supermatter/supermatter.dm]: Please mark your changes as explained here

**Review by ThatLing — DISMISSED** [review]

**Review by Amelia0010 — APPROVED** [review]

**Review by ThatLing — APPROVED** [review]

**Pull request merged**